sdk/eventhub/event-hubs/src/eventPosition.ts

```diff
@@ -63,7 +63,7 @@ export class EventPosition {
    * The same offset may refer to a different event as events reach the age limit for
    * retention and are no longer visible within the partition.
    */
-  offset?: string;
+  offset?: number | "@latest";
```

Comment:
I wonder if we can remove the `@latest` from the type here if the static method `latest()` is updated to 
` return new EventPosition({ offset: EventPosition.endOfStream });`
instead of
`return EventPosition.fromOffset(EventPosition.endOfStream);`

If this is possible, then our offset typing would get simplified